Clic&Farm-iOS.xcodeproj/project.pbxproj

```diff
@@ -748,7 +746,7 @@
 				GCC_WARN_UNINITIALIZED_AUTOS = YES_AGGRESSIVE;
 				GCC_WARN_UNUSED_FUNCTION = YES;
 				GCC_WARN_UNUSED_VARIABLE = YES;
-				IPHONEOS_DEPLOYMENT_TARGET = 11.4;
+				IPHONEOS_DEPLOYMENT_TARGET = 12.1;
```

Comment:
What does that implies? How much market % are we reaching if we're moving this from 11.4 to 12.1? Also, why exactly are we doing so?

Reply:
It implies user has to be on iOS 12.1 or above to install the app, this is not what we want. I think deployment targets updated itself when I updated Xcode to 10.1. Deployment target is now default's.